tests/test_modeling_common.py

```diff
@@ -812,7 +812,7 @@ def test_multigpu_data_parallel_forward(self):
             # Wrap model in nn.DataParallel
             model = torch.nn.DataParallel(model)
             with torch.no_grad():
-                _ = model(**inputs_dict)
+                _ = model(**self._prepare_for_class(inputs_dict, model_class))
```

Comment:
@sgugger added the prepare function here because otherwise longformer tests were failing